pkg/commands/engine/analyze.go

```diff
@@ -117,10 +118,11 @@ func CreateGetKeysCommand() *cobra.Command {
 
 func CreateEvalCommand() *cobra.Command {
 	return &cobra.Command{
-		Use:   "eval <measure 1> [<measure 2...>] by <dimension 1> [<dimension 2...]",
-		Args:  cobra.MinimumNArgs(1),
-		Short: "Evaluate a list of measures and dimensions",
-		Long:  `Evaluate a list of measures and dimensions. To evaluate a measure for a specific dimension use the <measure> by <dimension> notation. If dimensions are omitted then the eval will be evaluated over all dimensions.`,
+		Use:               "eval <measure 1> [<measure 2...>] by <dimension 1> [<dimension 2...]",
+		Args:              cobra.MinimumNArgs(1),
+		ValidArgsFunction: listFieldsForCompletion,
```

Comment:
I don't think `eval` should have this completion.
If `eval` should have completion, it has to be tailored to this command which is quite different from others.

Reply:
Yep lets keep that one for later